.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: Automatic PR for Development Update
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Extract version information
+        run: |
+          APP_VERSION=$(echo "${{ github.event.release.tag_name }}" | grep -oP '\d+\.\d+\.\d+')
+          echo "Extracted APP_VERSION: $APP_VERSION"
+          echo "APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
+
+          VERSION=$APP_VERSION  
+          echo "Extracted VERSION: $VERSION"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Update version in files
+        run: |
+          echo "set(APP_VERSION ${{ env.APP_VERSION }})" >> CMakeLists.txt
+          echo "VERSION = ${{ env.VERSION }}" >> src/mudlet.pro
```

Comment:
These two lines will append the versions to the "Makefiles". However this will not replace the current versions that are used within code and the files themselves. Rather, you'd have to __replace__ the current variable content at the position it is right now.